dwave/cloud/api/client.py

```diff
@@ -282,10 +282,6 @@ class CachingSession(VersionedAPISession):
             Force cache update, skipping time-based or etag-based validation.
     """
 
-    _cache_enabled = False
-    _maxage = None
-    _store = None
```

Comment:
Why remove these? It doesn't seem to change anything, but might cause some confusion when seeing them being accessed below. Not that it matters much, though.

Reply:
Removed them exactly to reduce confusion :smile: 

The line below defines the default config, which is different from these values. Sure, there are other ways to harmonize them, but by removing them I also wanted to make sure they're not used if cache is not configured (as they're meaningless then). IMO, only `_cache_enabled` would make sense to always be available, but it's set on construction anyway...

Reply:
I see your point. Why not remove their usage altogether since they seem to only used as temporary variables inside the `CachingSession` class?

Reply:
They're not temporary variables. `_maxage` is the default maxage used if maxage is not present in server's `Cache-Control` header. `_store` is the actual key/value cached data store.

Reply:
Ok, fair enough, that actually makes sense.